numpy/typing/tests/data/fail/twodim_base.pyi

```diff
@@ -5,11 +5,11 @@ import numpy.typing as npt
 
 
 def func1(ar: npt.NDArray[Any], a: int) -> npt.NDArray[np.str_]:
-    pass
+    ...
 
 
 def func2(ar: npt.NDArray[Any], a: float) -> float:
-    pass
+    ...
```

Comment:
Again that's not me, that's ruff:
```console
$ ruff check --select PYI009 numpy/typing/tests/data/fail/array_like.pyi --fix
Found 1 error (1 fixed, 0 remaining).
$ 
$ git diff
diff --git a/numpy/typing/tests/data/fail/array_like.pyi b/numpy/typing/tests/data/fail/array_like.pyi
index 133b5fd497..f14fa272d9 100644
--- a/numpy/typing/tests/data/fail/array_like.pyi
+++ b/numpy/typing/tests/data/fail/array_like.pyi
@@ -3,7 +3,7 @@
 
 
 class A:
-    pass
+    ...
 
 
 x1: ArrayLike = (i for i in range(10))  # E: Incompatible types in assignment
$ 
```

Reply:
I expect it is just replacing `pass`. Does it correct it if it is on the same line? IIRC, this turned up in the previous big PR, and an exception was added to Ruff(?) to allow the ellipses on the same line, as is the convention for annotations.

Reply:
Maybe it was Black that added that exception for pyi files. In any case, I'm willing to let this go, and if future formatters do the one line thing, we can make that change. The aim here is to get closer to automatic formatting.

Reply:
I was trying to avoid manual changes. The ruff formatter would probably move `...` inline, the linter just replaces `pass` with `...`. Since formatters such as black and ruff are currently not entirely compatible with numeric programming, I'll change that manually for the time being.